.github/workflows/libjs-test262.yml

```diff
@@ -50,14 +50,19 @@ jobs:
 
       - name: Install dependencies
         run: |
-          sudo apt-get update
-          sudo apt-get install -y ninja-build unzip gcc-13 g++-13 jq wget curl zip tar autoconf autoconf-archive automake nasm pkg-config libgl1-mesa-dev rsync
+          wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
+          sudo add-apt-repository 'deb http://apt.llvm.org/noble/ llvm-toolchain-noble-19 main'
```

Comment:
probably a follow-up, but using apt-key like this is deprecated.

This should be done like this:

```sh
    . /etc/lsb-release
    wget -O /usr/share/keyrings/llvm-snapshot.gpg.key https://apt.llvm.org/llvm-snapshot.gpg.key
    echo "deb [signed-by=/usr/share/keyrings/llvm-snapshot.gpg.key] http://apt.llvm.org/${DISTRIB_CODENAME}/ llvm-toolchain-${DISTRIB_CODENAME}-${LLVM_VERSION} main" | tee -a /etc/apt/sources.list.d/llvm.list
```

Reply:
Er I guess in Noble, they updated it so you're supposed to use these .sources files instead? Bit of a pain for CI scripts though

/etc/apt/sources.list.d/llvm-org.sources
```
Types: deb
URIs: http://apt.llvm.org/noble/
Suites: llvm-toolchain-noble-19
Components: main
Architectures: amd64
Signed-By: /etc/apt/keyrings/llvm-snapshot.gpg.key
```